src/styles/index.scss

```diff
@@ -1,3 +1,12 @@
+@import 'utils/variables';
+@import 'card';
```

Comment:
Ensure that the `card.scss` file is correctly imported and contains the styles for the card component.